packages/studio/src/workbench/EditorPane.jsx

```diff
@@ -84,7 +86,10 @@ export default observer(function EditorPane() {
         theme="vs-dark"
         height="100%"
         onChange={(e) => {
-          store.code.update(e, true);
+          if (debounceReference) clearTimeout(debounceReference);
+          debounceReference = setTimeout( () => {
+            store.code.update(e, true);
```

Comment:
You probably should clear the reference here as well? The best way might be to just use the `debounce` depedency that is[ part of the package anyway](https://github.com/sgenoud/replicad/blob/main/packages/studio/src/viewers/EditorViewer.jsx#L17).